deploy/docker/localkube-dind/Dockerfile

```diff
@@ -0,0 +1,71 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:jessie
+
+ARG CNI_RELEASE=0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff
```

Comment:
In attempting to use the github release I am getting errors:
E0816 20:28:23.545093      17 pod_workers.go:182] Error syncing pod 6b416451-82c1-11e7-8b83-0242c
0a80902 ("kubernetes-dashboard-lkmpp_kube-system(6b416451-82c1-11e7-8b83-0242c0a80902)"), skippin
g: failed to "KillPodSandbox" for "6b416451-82c1-11e7-8b83-0242c0a80902" with KillPodSandboxError
: "rpc error: code = 2 desc = NetworkPlugin cni failed to teardown pod \"kubernetes-dashboard-lkm
pp_kube-system\" network: failed to find plugin \"bridge\" in path [/opt/bridge/bin /opt/cni/bin]
"
I think I will use the previous method for now